int32[] set_buttons
int32[] button_value
---
int32 code
string error_message
